chore(maps): change cluster url

ShahAnuj2610 opened the pull request:
## What does this PR do?
- migrate cluster to https://IkwcRqior:cda6348c-37c9-40f6-a144-de3cb18b57a0@arc-cluster-appbase-tryout-k8dsnj.searchbase.io

## Test
TBD

ShahAnuj2610 committed:
chore(maps): change cluster url

## What does this PR do?
- migrate cluster to https://IkwcRqior:cda6348c-37c9-40f6-a144-de3cb18b57a0@arc-cluster-appbase-tryout-k8dsnj.searchbase.io

## Test
TBD

bietkul reviewed (changes requested):
@ShahAnuj2610 Please update credentials.

ShahAnuj2610 committed:
chore(web): update credentials

## What does this PR do?
- change write credentials to read credentials

## Test
TBD

ShahAnuj2610 commented:
> @ShahAnuj2610 Please update credentials.

@bietkul done
